src/map/status.cpp

```diff
@@ -11139,10 +11145,15 @@ int32 status_change_start(struct block_list* src, struct block_list* bl,enum sc_
 			if (map_flag_gvg2(bl->m) || map_getmapflag(bl->m, MF_BATTLEGROUND)) val4 *= 5;
 			break;
 		case SC_CLOAKING:
-			if (!sd) // Monsters should be able to walk with no penalties. [Skotlex]
+			if (sd == nullptr) {
+				// Monsters have no walk penalties and the status change lasts for 10 seconds
```

Comment:
```suggestion
				// Non-players have no walk penalties and the status change lasts for 10 seconds
```

Reply:
Actually good point, I should check how other types behave here.